flint/flint-spark-integration/src/main/scala/org/apache/spark/sql/flint/config/FlintConfig.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.apache.spark.sql.flint.config
+
+import org.apache.spark.internal.config.ConfigReader
+
+/**
+ * Similar to SPARK ConfigEntry. ConfigEntry register the configuration which can not been
+ * modified.
```

Comment:
Does this mean the value can still be modified once configured, but config itself such as default value cannot be modified?